src/OSPSuite.InstallationValidator.Core/Domain/StartableProcess.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace OSPSuite.InstallationValidator.Core.Domain
+{
+   public class StartableProcess
+   {
+      private readonly Process _process;
+      private bool _exited;
+
+      public StartableProcess(string filePath, string arguments = null)
```

Comment:
what about passing a param arrays of arguments and this class is responsible for buiding the argument string? Would that be helpful?